tldw_Server_API/app/core/TTS/adapters/pocket_tts_cpp_runtime.py

```
@@ -0,0 +1,634 @@
+        raise TTSModelNotFoundError(
+            f"PocketTTS.cpp binary is not executable at {binary_path}",
+            provider=PROVIDER_KEY,
+            details={"binary_path": str(binary_path)},
+        )
+
+    if not tokenizer_path.exists():
+        raise TTSModelNotFoundError(
+            f"PocketTTS.cpp tokenizer not found at {tokenizer_path}",
+            provider=PROVIDER_KEY,
+            details={"tokenizer_path": str(tokenizer_path)},
+        )
```

Ruff (0.15.7)

[warning] Avoid specifying long messages outside the exception class

(TRY003)